src/utils.js

```diff
@@ -8,3 +8,29 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findLowest = (...numbers) => {
+  let lowest = Infinity;
+  let res = numbers.every((element) => {return Number.isInteger(element);});
+  try {
+    if(numbers.length < 1) {
+      throw "This function expects at least one argument";
+    }else if(!res) {
+      throw "The function expects an integer";
+    }
+    else {
+      numbers.forEach(number => {
+        if(number < lowest) {
+          lowest = number;
+        };
+      });
+      return lowest;
+    }
+  }
+  catch (err) {
+    return err;
+  }
+  
```

Comment:
- Let's apply the concept of early return here too
- let us default the lowest number to the first number in the array instead of `Infinity`

Reply:
ok